src/jl_uv.c

```diff
@@ -49,6 +49,16 @@ extern "C" {
 
 static uv_async_t signal_async;
 
+#ifdef _OS_WINDOWS_
+// uv_async_t is buggy on windows. Initializing one breaks the sysimg build.
```

Comment:
other than working around this pipe redirection thing, what consequences will making these no-ops have?